mmv1/third_party/terraform/services/alloydb/resource_alloydb_backup_test.go

```diff
@@ -53,41 +52,58 @@ resource "google_alloydb_backup" "default" {
 
   description = "example description"
   labels = {
-    "label" = "updated_key"
-    "label2" = "updated_key2"
+    "label" = "key"
   }
   depends_on = [google_alloydb_instance.default]
 }
 
 resource "google_alloydb_cluster" "default" {
   cluster_id = "tf-test-alloydb-cluster%{random_suffix}"
   location   = "us-central1"
-  network    = google_compute_network.default.id
+  network    = data.google_compute_network.default.id
 }
 
 resource "google_alloydb_instance" "default" {
   cluster       = google_alloydb_cluster.default.name
   instance_id   = "tf-test-alloydb-instance%{random_suffix}"
   instance_type = "PRIMARY"
+}
+
+data "google_compute_network" "default" {
+  name = "%{network_name}"
+}
+`, context)
+}
+
+// Updates "label" field
+func testAccAlloydbBackup_update(context map[string]interface{}) string {
```

Comment:
is this updating labels change intended here?

Reply:
Right, the labels updating test exists previously.